src/models/Users.js

```diff
@@ -0,0 +1,63 @@
+import mongoose from 'mongoose';
+
+const usersSchema = {
```

Comment:
@gracelungu a Model represents a single record that is why they should be singular